website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,288 @@
+# Operator Scope
+
+## Overview
+
+A `namespace-scoped` operator watches and manages resources in a single namespace, whereas a `cluster-scoped` operator watches and manages resources cluster-wide. 
```

Comment:
Meta note: we should consider writing documents with short-line paragraphs (100 characters or less per line) for new or re-formatted documents. They will be easier to review. This has been suggested a few times already.

Reply:
I overlook the long lines, but they are a pain to read IMO :)